src/lib/Scenes/Collection/Screens/CollectionHeader.tsx

```diff
@@ -9,33 +12,52 @@ interface CollectionHeaderProps {
   collection: CollectionHeader_collection
 }
 
+const HEADER_IMAGE_HEIGHT = 204
+
 export const CollectionHeader: React.SFC<CollectionHeaderProps> = props => {
-  const { title, image, headerImage } = props.collection
-  const url = headerImage ? headerImage : image.edges[0].node.imageUrl
+  const { title, image, headerImage, descriptionMarkdown: collectionDescription } = props.collection
+  const url = headerImage ? headerImage : image.edges[0]?.node.image.resized.url
```

Comment:
Given that [the type definition indicates that each of the properties in this chain are nullable](https://github.com/artsy/emission/pull/2035/files#diff-795092f79bb6cdf58c90c195769cd8ebR11), we should use `?.` through this whole chain: 

```
  const url = headerImage ? headerImage : image.edges[0]?.node?.image?.resized?.url
```

In addition, if the `OpageImageView` component on line 25 doesn't support `undefined` as a value for `imageURL`, we should assign a default value to url that *is* supported by `OpaqueImageView`. (By adding `|| "a-reasonable-default-value"` at the end of the line)

Reply:
Ah I literally just pushed this change as you were reviewing, I think its safeguarded now since it just defaults to an empty string.

Reply:
But it looks like you read my mind in https://github.com/artsy/emission/pull/2035/commits/2cde92478f119a5126cf4fd17204da3656c9e3f7 😄